hooks/useServiceWorker.ts

```
@@ -17,7 +17,14 @@ export function useServiceWorker() {
     }
 
     // In development, Serwist is disabled via next.config.ts
+    // Also unregister any stale SWs from previous production builds
     if (process.env.NODE_ENV === "development") {
+      navigator.serviceWorker.getRegistrations().then((registrations) => {
+        registrations.forEach((reg) => reg.unregister());
```

Biome (2.4.6)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)